app.js

```diff
@@ -53,8 +53,7 @@ const loadConfig = () => {
   try {
     fs.accessSync(argv.config, fs.F_OK);
   } catch (e) {
-    log(['error'], `Unable to access config directory ${argv.config}`);
-    log(e);
+    log(['error'], e);
```

Comment:
Just confirming this will log the actual error. I've had issues stringifying errors